code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -947,12 +1069,12 @@
 	reagent_state = SOLID
 	taste_description = "iron"
 	pH = 6
-
+	overdose_threshold = 30
 	color = "#c2391d"
 
 /datum/reagent/iron/on_mob_life(mob/living/carbon/C)
```

Comment:
Surely an efficacy reduction by 98% on top of an OD is harsh? Taking 30u would only yield 0.75u total after 2.5 minutes of metabolism. In this case sugar is a better blood treatment. What options are there for people to treat blood loss when medbay ceases to be an option?

Reply:
Cargo can order synthetic blood bags, and they can be administered with the CMO hypo in single, large doses if needed.

Reply:
*laughs in bloody mary*